src/playdate/graphics.nim

```diff
@@ -11,22 +11,49 @@ export graphics
 {.hint[DuplicateModuleImport]: off.}
 import bindings/graphics {.all.}
 
-type LCDBitmapObj = object of RootObj
-    resource {.requiresinit.}: LCDBitmapPtr
-    free: bool
+type
+    LCDBitmapObj = object of RootObj
+        res {.requiresinit.}: LCDBitmapPtr
+
+    LCDBitmapObjRef = ref LCDBitmapObj
+
+    LCDBitmap* = object
+        case managed: bool
+        of true:
+            obj: LCDBitmapObjRef
+        of false:
+            res: LCDBitmapPtr
+
 proc `=destroy`(this: var LCDBitmapObj) =
     privateAccess(PlaydateGraphics)
-    if this.free:
-        playdate.graphics.freeBitmap(this.resource)
-type LCDBitmap* = ref LCDBitmapObj
+    if this.res != nil:
+        playdate.graphics.freeBitmap(this.res)
+
+proc `=copy`(a: var LCDBitmapObj, b: LCDBitmapObj) {.error.}
+
+proc newUnmanagedBitmap(point: LCDBitmapPtr): auto =
```

Comment:
Can you rename this to `bitmapPointer`. Very sdtrong associations with a 2d point here.

Same for newManagedBitmap